deps/openlibm.version

```diff
@@ -1,2 +1,2 @@
-OPENLIBM_BRANCH=v0.4.1
-OPENLIBM_SHA1=3ee2a6e6a9fc68cbf2ab00f830b9059215e40b06
+OPENLIBM_BRANCH=master
```

Comment:
Possibly. I can't really read what the git externals makefile macros are doing. It's only used in one or two minor places. More saying for the sake of making release tags for packagers to be aware of as opposed to picking some random commit hash. Also note the VERSION and SONAME look out of date on master of both openlibm and openspecfun. We need a better way of enforcing that those get updated for releases.